styles/GlobalStyles.tsx

```diff
@@ -0,0 +1,98 @@
+import { createGlobalStyle } from 'styled-components';
+
+// Max Width is divided into 320 blocks. So it breaks every 320px
+// This works for the smaller iPhones so seems like a good idea :D
+//
+// This should general break up into 300px blocks with 10px spacing
+// on each side
+//
+// Wrapping 10px on the outside might need consideration
+//
+// Smallest = 320px
+// Medium = 640px
+// Large = 960px
+// FullSize = 1280px
+const GlobalStyles = createGlobalStyle`
+  html {
+    font-size: 62.5%; /* Now 10px = 1rem! */
+  }
+  body {
+    font-size: 20px; /* px fallback */
+    font-size: 2rem; /* default font-size for document */
+    line-height: 1.5; /* a nice line-height */
+    font-family: 'Raleway', sans-serif;
+    background-color: #ffb700;
+    color: #ffffff;
+  }
+  h1,h2,h3,h4,h5,h6{
+    font-family: 'Fredoka One', cursive;;
+    letter-spacing: 0.2rem;
+    line-height: 1.1;
+    margin: 1rem 0;
```

Comment:
This is where some rouge margin is coming from

Reply:
I read that thinking 🤔 rouge?? red in french?? then realised you meant rogue!